src/tests/fp.test.ts

```
@@ -0,0 +1,140 @@
+  filter,
+  reduce,
+  identity,
+  constant,
+} from '../core/fp';
+
+describe('Functional Programming Utilities', () => {
+  describe('pipe', () => {
+    it('should pipe a single value through multiple functions', () => {
+      const add2 = (x: number) => x + 2;
+      const multiply3 = (x: number) => x * 3;
+      const toString = (x: number) => x.toString();
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)